topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorBaseIT.java

```diff
@@ -404,7 +405,7 @@ protected void awaitTopicConfigInKube(String resourceName, String key, String ex
         waitFor(() -> {
             KafkaTopic topic = operation().inNamespace(NAMESPACE).withName(resourceName).get();
             LOGGER.info("Polled topic {}, waiting for config change", resourceName);
-            String gotValue = TopicSerialization.fromTopicResource(topic).getConfig().get(key);
+            String gotValue = TopicSerialization.fromTopicResource(topic, "3.0.0").getConfig().get(key);
```

Comment:
I think having the version hardcoded here will cause issues int he future. Applies also to the one above and below.